crates/chain/Cargo.toml

```diff
@@ -20,12 +20,15 @@ serde_crate = { package = "serde", version = "1", optional = true, features = ["
 # Use hashbrown as a feature flag to have HashSet and HashMap from it.
 hashbrown = { version = "0.9.1", optional = true, features = ["serde"] }
 miniscript = { version = "11.0.0", optional = true, default-features = false }
+async-trait = {  version = "0.1.80", optional = true }
 
 [dev-dependencies]
 rand = "0.8"
 proptest = "1.2.0"
 
 [features]
-default = ["std", "miniscript"]
+default = ["std", "miniscript", "persist"]
 std = ["bitcoin/std", "miniscript?/std"]
 serde = ["serde_crate", "bitcoin/serde", "miniscript?/serde"]
+persist = ["miniscript"]
```

Comment:
I don't think we should have a `persist` feature. Instead, we disable `CombinedChangeSet` when we don't have `miniscript`.